recipe_batches/recipe_batches.py

```diff
@@ -1,10 +1,26 @@
 #!/usr/bin/python
 
 import math
+import operator
 
 def recipe_batches(recipe, ingredients):
-  pass 
-
+  # Need to get the max value from ingredients.items, this is a starting point
+  batch = max(ingredients.items(), key=operator.itemgetter(1))[1]
+  # Now I need to loop 
+  for recKey in recipe.items():
+    # if the key from recipes does not appear in ingredients, batch=0 because we can't make recipe
+    if recKey[0] not in ingredients.keys():
+      batch = 0
```

Comment:
if `batch` is already set to 0, you can break out of the loop. no matter what other combinations there are, if there's not enough of one ingredient then you can make no batches. once you know that, you can simply return 0.